.github/workflows/test.yaml

```diff
@@ -28,18 +28,19 @@ jobs:
       fail-fast: false
       matrix:
         include:
-          - ckan-container-version: "2.9-py2"
+          - ckan-container-version: "2.9.7-py2"
             ckan-postgres-version: "2.9"
             ckan-solr-version: "2.9-solr8"
             ckan-requirements: "requirements.py2.txt"
-          - ckan-container-version: "2.9"
+          - ckan-container-version: "2.9.7"
             ckan-postgres-version: "2.9"
             ckan-solr-version: "2.9-solr8"
             ckan-requirements: "requirements.py3.txt"
-          - ckan-container-version: "2.10"
-            ckan-postgres-version: "2.10"
-            ckan-solr-version: "2.10"
-            ckan-requirements: "requirements.py3.txt"
+            # package_activity_list not found in ckan 2.10 ?
```

Comment:
uncomment the ckan 2.10 tests. We're keeping them in all projects to keep in mind future upgrade to new ckan version.